src/status_im/ui/screens/views.cljs

```diff
@@ -67,13 +67,18 @@
             modal-view [:get :modal]]
     (when view-id
       (let [current-view (validate-current-view view-id signed-up?)]
-        (let [component (case current-view
-                          (:home :wallet :my-profile) main-tabs
+        (let [current-view' (case current-view
+                             (:home :wallet :my-profile)
+                             :main-tabs
+
+                             current-view)
+              component (case current-view'
+                          :main-tabs main-tabs
                           :wallet-send-transaction send-transaction
                           :wallet-transaction-sent transaction-sent
                           :choose-recipient choose-recipient
                           :wallet-request-transaction request-transaction
-                          (:transactions-history :unsigned-transactions) wallet-transactions/transactions
```

Comment:
What does this change have to do with navigation?

Reply:
again rebasing, will fix this